code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -113,6 +113,13 @@
 		reagents.reaction(user.loc)
 		reagents.clear_reagents()
 
+/obj/item/reagent_containers/spray/empty
```

Comment:
Please rename this to something that better implies this is just a nerfed spraybottle. Maybe `manufactured` or something?

Reply:
Erm... It is nerfed by default now. It is just spray bottle, like cleaner, but empty. If you fill it with cleaner it will have no difference to /obj/item/reagent_containers/spray/cleaner
I still dont't know why it should be nerfed, but... This is how it is. I'm not making spray bottles normal again, after nerf.

Reply:
I just mean code-wise, it'd be nice if this had a different type to avoid confusion down the line/people accidentally mapping this in by accident.

Reply:
I just think that "empty" is more clear and suitable. It is literally general spray bottle, but empty...